Random.py

```diff
@@ -46,21 +46,4 @@ def print_students(self):
 manager = StudentManager()
```

Comment:
The indentation on lines 38-46 is inconsistent. It should align with the function definition. ```diff -      if direction == "up": +    if direction == "up": -         y -= 1 +        y -= 1 -      elif direction == "down": +    elif direction == "down": -         y += 1 +        y += 1 -      elif direction == "left": +    elif direction == "left": -         x -= 1 +        x -= 1 -      elif direction == "right": +    elif direction == "right": -         x += 1 +        x += 1 ``` ---